amostra/tests/test_openapi.py

```diff
@@ -0,0 +1,7 @@
+import json
+
+
+def test_load():
+    with open('../../docs/source/openapi.json') as f:
```

Comment:
This path will be evaluated relative to the current working directory of the Python process. That is, its meaning will depend on which directory you are sitting in when you run `pytest`. A common pattern to avoid this problem is to do:

```py
here = os.path.abspath(os.path.dirname(__file__)); print(repr(here))
# then os.path.join(here, '../../docs/source/openapi.json')
```